config

```diff
@@ -110,6 +110,8 @@
 # Value: debug | info | warning | error | critical
 #level = warning
 
+logfile = /var/log/radicale.log
```

Comment:
this can trigger SELinux issues, better would be /var/log/radicale/radicale.log + a proper logwatch configuration for rotation, in case no autorotation is implemented